website/content/en/docs/deployment/cognito-rds-s3/guide.md

```diff
@@ -54,11 +54,15 @@ Refer to the [general prerequisites guide](/kubeflow-manifests/docs/deployment/p
         # Kubeflow Istio Resources
         kustomize build upstream/common/istio-1-9/kubeflow-istio-resources/base | kubectl apply -f -
         
-        # KFServing
+        # Kserve
+        kustomize build upstream/contrib/kserve/kserve | kubectl apply -f -
```

Comment:
why is this different from `awsconfigs/apps/kserve` in the deployments folder manifests?